src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace System.Linq.Expressions.Interpreter
 {
+#if FEATURE_MAKE_RUN_METHODS
```

Comment:
Are these flags new or exist?  Under what circumstances are they set?  If this is dead code, should it just be deleted instead?